### DISCUSSION TITLE
Removing unused components of the extension

### DISCUSSION THREAD

**Discussion by dandv** [Jonghakseo/chrome-extension-boilerplate-react-vite#812] [Q&A]
Thanks for creating the Discussions @Jonghakseo !

I've been meaning to ask, what is the procedure to remove components of an extension thar aren't used, for example the DevTools in most cases, and many others, the New Tab?

:pray:

**Comment by Jonghakseo** [comment]
The fundamental step in excluding unused components is to remove them from the manifest. However, components still be included in bundled files, and you'll need to remove the actual files to remove them from the build pipeline. For example, you can clear devtools, devtools-panel inside the page folder.

**Comment by PatrykKuniczak** [comment]
We'll create solution for that next year, we have meeting about it a months ago

**Comment by Antroll** [comment]
~~I've removed the new tab example from the code and manifest, but for some reason (that I can not find), the new tab in the browser opens with an empty page. Please help to avoid this behavior.~~

A full rebuild helped to resolve the issue.

**Comment by PatrykKuniczak** [comment]
> I've removed the new tab example from the code and manifest, but for some reason (that I can not find), the new tab in the browser opens with an empty page. Please help to avoid this behavior.

You need to share us minimal code repo

**Comment by Jonghakseo** [comment]
- #875
